rubrik_cdm/data_management.py

```diff
@@ -1559,16 +1588,16 @@ def _validate_sql_recovery_point(self, mssql_id, date, time, timeout=30):  # pyl
         recovery_point = {}
         recovery_point['is_recovery_point'] = is_recovery_point
         recovery_point['recovery_timestamp'] = recovery_timestamp
-        
+
         return recovery_point
 
     def sql_instant_recovery(self, db_name, date, time, sql_instance=None, sql_host=None, finish_recovery=True, max_data_streams=0, timeout=30):  # pylint: ignore
         """Perform an instant recovery for MSSQL database from a specified recovery point.
 
         Arguments:
             db_name {str} -- The name of the database to instantly recover.
-            date {str} -- The recovery_point date to recover to formated as `Month-Day-Year` (ex: 1-15-2014). 
-            time {str} -- The recovery_point time to recover to formated as `Hour:Minute AM/PM` (ex: 1:30 AM).
+            date {str} -- The recovery_point date to recover to formated as `Month-Day-Year` (ex: 1-15-2014).
+            time {str} -- The recovery_point time to recover to formated formated as `Hour:Minute AM/PM` (ex: 1:30 AM).
```

Comment:
Duplicate word formated `time {str} -- The recovery_point time to recover to formated formated as ...`